README.md

```diff
@@ -251,9 +258,9 @@ export default defineSchema({
 
 Be sure this is your default export from this file, we'll validate the schema and build out the GraphQL API with it.
 
-### `sections`
+### `collections`
 
-The top-level key in the schema is an array of sections, a `section` informs the API about _where_ to save content. You can see from the example that a `pages` document would be stored in `content/pages`, and it can be the shape of any `template` from the `templates` key.
+The top-level key in the schema is an array of _collections_, a `collection` informs the API about _where_ to save content. You can see from the example that a `pages` document would be stored in `content/pages`, and it can be the shape of any `template` from the `templates` key.
```

Comment:
I wonder if the example schema here should avoid referencing a `pages` collection in the same way the starter's `schema.ts` was changed.